flink-scala/src/main/scala/org/apache/flink/api/scala/ExecutionEnvironment.scala

```diff
@@ -64,15 +62,19 @@ import scala.reflect.ClassTag
  *  created. If the program is submitted to a cluster a remote execution environment will
  *  be created.
  */
-class ExecutionEnvironment(javaEnv: JavaEnv) {
+class ExecutionEnvironment(javaEnv: JavaEnv) extends AbstractExecutionEnvironment {
 
   /**
-   * @return the Java Execution environment.
+   * Returns the enclosed Java ExecutionEnvironment for special use cases.
+   *
+   * @return reference to the ExecutionEnvironment of the Java API
    */
   def getJavaEnv: JavaEnv = javaEnv
+
   /**
    * Gets the config object.
    */
+  @Override
```

Comment:
In Scala override is done as:
```
override def getConfig ...
```

this is also in StreamExecutionEnvironment.scala